From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Andrew Kaster <akaster@serenityos.org>
Date: Sat, 3 Dec 2022 09:08:11 -0700
Subject: [PATCH] Serenity: Disable local domain name lookup via resolv

Serenity doesn't support /etc/resolv.conf or the structures in
<resolv.h> to do Unix-like domain name resolution.
---
 src/network/kernel/qhostinfo_unix.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/network/kernel/qhostinfo_unix.cpp b/src/network/kernel/qhostinfo_unix.cpp
index 12d8c04d104b56d3c8d77d338c6427604479003c..f03af1c2b24468fe3a8f5aee952c239128949b03 100644
--- a/src/network/kernel/qhostinfo_unix.cpp
+++ b/src/network/kernel/qhostinfo_unix.cpp
@@ -162,7 +162,7 @@ QHostInfo QHostInfoAgent::fromName(const QString &hostName)
 
 QString QHostInfo::localDomainName()
 {
-#if !defined(Q_OS_VXWORKS) && !defined(Q_OS_ANDROID)
+#if !defined(Q_OS_VXWORKS) && !defined(Q_OS_ANDROID) && !defined(Q_OS_SERENITY)
     resolveLibrary(NeedResNInit);
     if (local_res_ninit) {
         // using thread-safe version
